paddle/fluid/framework/new_executor/new_executor_defs.cc

```diff
@@ -693,6 +693,29 @@ const std::vector<size_t>& Instruction::GCCheckVars() const {
   return gc_check_var_list_;
 }
 
+void Instruction::InferNeedStreamSyncVars() {
+  auto ParseIndexFromMap = [](
```

Comment:
Why use lambda function here?

Reply:
This code has been deleted, please check the new commit.